core/validation/failures.py

```diff
@@ -307,6 +316,9 @@ def serialise_user_centered_failures(validation_failures: list[ValidationFailure
         for failure in validation_failures
         if isinstance(failure, TFUCFailureMessage)
     ]
+    # one pre-transformation failure means payload is entirely pre-transformation failures
+    if any(failure in PRETRANSFORMATION_FAILURE_UC_MESSAGE_BANK.values() for failure in uc_failures):
+        return uc_failures
```

Comment:
Another consideration for this code is that we expect the output to be in the form:
```
            "{"
            '    "detail": "Workbook validation failed",'
            '    "status": 400,'
            '    "title": "Bad Request",'
            '    "validation_errors": {'
            '        "PreTransformationErrors": ['
            '            "The selected file must be a CSV"'
            "        ]"
            "    }"
            "}"
```
as per the Submit unit test mocks.

This would currently return along the lines of
```
            b"{"
            b'    "detail": "Workbook validation failed",'
            b'    "status": 400,'
            b'    "title": "Bad Request",'
            b'    "validation_errors": {'
            b'        "TabErrors": ['
            b'            "The selected file must be a CSV"'
            b"        ]"
            b"    }"
            b"}"
```
as per the commented out code in `errors.py`

Reply:
Agree with the above and have implemented.

Also expanded the comment to account for PreTransformationFailures.